Jint/Runtime/Debugger/BreakPoint.cs

```diff
@@ -1,32 +1,20 @@
 ﻿namespace Jint.Runtime.Debugger
 {
-    public sealed class BreakPoint
+    // BreakPoint is not sealed. It's useful to be able to add additional properties on a derived BreakPoint class (e.g. a breakpoint ID
+    // or breakpoint type) but still let it be managed by Jint's breakpoint collection.
+    public class BreakPoint
```

Comment:
is there something deriving from this in test project, would make sure nobody (like me) would accidentally seal again, as many changes already could also do the namespace and nullable thing